content/W2015/W15-event-profsdinner.md

```diff
@@ -0,0 +1,23 @@
+Title: Dinner With Profs
+Date: 2015-03-26 5:30PM
+Category: Events
+Tags: profs, dinner
+Slug: dinner-with-profs
+Author: Anna Lorimer
+Summary: WiCS and WiM are hosting a Dinner with Profs event! Come get to know other successful women in your field. 
+
+WiCS and WiM would like to invite all undergraduate ciswomen and transwomen, and non-binary folk to our Dinner with Profs event!
+
+This event is an opportunity for ciswomen and transwomen, and non-binaryfolk to in Math and CS to enjoy a semi-formal dinner while getting to know other successful people in the same fields.
+It promises to be a great event! 
```

Comment:
I bet you it's wrapping on your screen but not actually inserting the necessary newlines. Will need to find a way to do that!